px-view-header.html

```diff
@@ -40,23 +40,23 @@
 <dom-module id="px-view-header">
   <template>
     <style include="px-view-header-styles"></style>
-    
-    <div id="container" class="header flex flex--justify">
-      <div id="left" class="flex flex--col flex--center">
+
+    <div id="container" class="header">
+      <div id="left">
         <content select=".left"></content>
```

Comment:
I realize this didn't change but it the `.` in `.left` correct for content tags?